firebase-database/src/androidTest/java/com/google/firebase/database/TransactionTest.java

```diff
@@ -568,6 +567,7 @@ public void onComplete(
 
     // Note that the set actually raises two events, one overlaid on top of the original transaction
     // value, and a second one with the re-run value from the third transaction
+    assertEquals(nodeFooSnaps.size(), 4); // three transactions with one redo
```

Comment:
This variable was being written to but never read from. I added an arbitrary test though we could also just remove its pushes.